impl/src/main/java/org/jboss/arquillian/performance/event/PerformanceAnalyzer.java

```diff
@@ -139,14 +140,29 @@ public boolean accept(File pathName)
         }
         return prevResults;
     }
+    
+    private File getArqPerfFolder() {
+    	
+    	String arqPerfFolder = System.getProperty(ARQ_PERF_FOLDER);
```

Comment:
This should be moved to normal Arquillian Configuration handling. Inject an instance of ArquillianDescriptor and find a Extension named performance. Then read the value from there. Core will handle the possible system properties / env override etc.